tests_v2/test_browser_api_comprehensive.py

```
@@ -139,23 +137,65 @@ def _build_pages() -> tuple[str, str]:
 """.strip()
 
-    return _data_url(page1), _data_url(page2)
+    return {
+        "/probe.html": "<!doctype html><html><body><button id='probe'>P</button></body></html>",
+        "/cmux-browser-comprehensive-1.html": page1,
+        "/cmux-browser-comprehensive-2.html": page2,
+    }
+
+
+@contextlib.contextmanager
+def _serve_pages(pages: dict[str, str]):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_serve_pages`

(ANN202)